docs/source/user_guide/extraction.md

```diff
@@ -1,3 +1,126 @@
 # Extracting IDPs from pipeline derivatives
 
-Work in progress!
+Extraction pipelines pick up where processing pipelines left off: they transform preprocessing outputs into analysis-ready files containing imaging-derived phenotypes (IDPs; e.g., feature tables, connectivity matrices).
+
+Just like with the BIDS conversion and processing pipelines, Nipoppy uses the {term}`Boutiques framework <Boutiques>` to run extraction pipelines.
+
+<!--
+By default, new Nipoppy datasets (as created with [`nipoppy init`](<project:../cli_reference/init.md>)) are populated with descriptor files and default invocation files for the following processing pipelines:
+- TODO
+-->
+
+## Summary
+
+### Prerequisites
+
+- A Nipoppy dataset with a valid global configuration file and an accurate manifest
+    - See the [Quickstart guide](../quickstart.md) for instructions on how to set up a new dataset
+- Processed imaging data in {{dpath_pipeline_output}} for the relevant processing pipeline(s) that the extractor depends on
+    - See <project:processing.md>
+
+<!--
+```{include} ./inserts/apptainer_stub.md
+```
+-->
+
+### Data directories
+
+| Directory | Content description |
+|---|---|
+| {{dpath_pipeline_output}} | **Input** -- {{content_dpath_pipeline_output}} |
+| {{dpath_pipeline_idp}} | **Output** -- {{content_dpath_pipeline_idp}} |
+
+### Commands
+
+- Command-line interface: [`nipoppy extract`](<project:../cli_reference/extract.md>)
+- Python API: {class}`nipoppy.workflows.ExtractionRunner`
+
+### Workflow
+
+1. Nipoppy will check the {term}`imaging bagel file` and loop over all participants/sessions that have completed processing for all the pipelines listed in the `PROC_DEPENDENCIES` field.
+    - See <project:tracking.md> for more information on how to generate the bagel file
+2. For each participant-session pair:
+    1. The pipeline's invocation will be processed such that template strings related to the participant/session and dataset paths are replaced by the appropriate values
+    2. The pipeline is launched using {term}`Boutiques`, which will be combine the processed invocation with the pipeline's descriptor file to produce and run a command-line expression
+
+## Configuring extraction pipelines
+
+Just like with BIDS pipelines and processing pipelines, pipeline and pipeline step configurations are set in the global configuration file (see [here](./global_config.md) for a more complete guide on the fields in this file).
+
+There are several files in pipeline step configurations that can be further modified to customize pipeline runs:
+- `INVOCATION_FILE`: a {term}`JSON` file containing key-value pairs specifying runtime parameters. The keys correspond to entries in the pipeline's descriptor file.
+
+```{note}
+By default, pipeline files are stored in {{dpath_pipelines}}`/<PIPELINE_NAME>-<PIPELINE_VERSION>`.
```

Comment:
Decided that this was not super relevant since there is no section on "how to write an extraction pipeline", so will not include it in this PR. But some example guidelines would be:
- Name the extraction pipeline something useful (e.g. not `freesurfer_extractor`)
- Version it independently of the associated processing pipelines